.github/ISSUE_TEMPLATE/1_dont_ignore.yml

```diff
@@ -0,0 +1,21 @@
+name: "README: DO NOT IGNORE OR DELETE THE ISSUE TEMPLATE"
```

Comment:
Do we really need that? I would minimize the amount of bloat / text that is thrown at user, so they can focus on important points and instructions instead of boilerplate warnings.

Reply:
one can't really answer this question, we don't know how many unnecessary issues this warning prevented.

there aren't really instruction yet. if one gets confused by this simple warning, i believe the problems isn't really on our side or with those templates.

though if we implement what @na-na-hi suggested with the checkboxes at the end of the issue templates, this is imo completely useless and redundant.

Reply:
> one can't really answer this question, we don't know how many unnecessary issues this warning prevented.

I don't have the numbers, obviously. But overstimulating users with too much information will result in them ignoring it. The same as TOS everywhere, no one reads it, because it is too much.

> though if we implement what @na-na-hi suggested with the checkboxes at the end of the issue templates, this is imo completely useless and redundant.

I think checkboxes is a good idea and makes user actively think about each of the item separately. And can go back if anything was missing.